src/VirtoCommerce.Platform.Web/wwwroot/js/app/modularity/blades/modules-list.js

```diff
@@ -59,7 +59,13 @@ function ($scope, bladeNavigationService, uiGridConstants, uiGridHelper, moduleH
                     executeMethod: function () { $scope.confirmActionInDialog('uninstall', $scope.gridApi.selection.getSelectedRows()); },
                     canExecuteMethod: installedItemsChecked,
                     permission: 'platform:module:manage'
-                }
+                },
+                {
+                    name: "platform.commands.grouping", icon: 'fas fa-cubes',
```

Comment:
The problem is that when you click "Show bundles" and see them, you need to click "Show bundles" again to hide.  And it can be a little confusing. The best way is to change somehow the title of toolbar button when you click on it (show/hide). The simple way - choose another more neutral title - just "Bundles" for example.

Reply:
I will change to Hide bundles